app/scripts/components/exploration/components/datasets/colorRangeSlider.tsx

```diff
@@ -0,0 +1,270 @@
+            inputError.min || inputError.largerThanMax
+              ? 'border-secondary-vivid text-secondary-vivid'
+              : ' border-base-light'
+          }`}
+          value={minValRef.current}
+          onChange={(event) => {
+            minValRef.current = event.target.value;
+            const value = Number(event.target.value);
+
+            if (value > maxVal - minMaxBuffer)
+              return setInputError({ ...inputError, largerThanMax: true });
+            if (value < min || value > max) {
+              return setInputError({ ...inputError, min: true });
+            } else {
+              setInputError({
+                ...inputError,
+                min: false,
+                largerThanMax: false
+              });
+              setMinVal(Math.min(value, maxVal - minMaxBuffer));
+            }
+          }}
+        />
+
+        <div className='position-relative container display-flex flex-align-center flex-justify-center padding-x-2'>
+          <input
+            type='range'
+            min={min}
+            max={max}
+            step='.001'
+            value={minVal}
+            onChange={(event) => {
+              const value = Math.min(
+                Number(event.target.value),
+                maxVal - minMaxBuffer
+              );
+              resetErrorOnSlide(value, 'min');
+              setMinVal(value);
+              minValRef.current = value;
+            }}
+            className={`thumb ${thumbPosition} z-index-300`}
+            style={{
+              zIndex: minVal >= max - 10 * minMaxBuffer ? '500' : '300'
+            }}
+          />
+          <input
+            type='range'
+            min={min}
+            step='.001'
+            max={max}
+            value={maxVal}
+            onChange={(event) => {
+              const value = Math.max(
+                Number(event.target.value),
+                minVal + minMaxBuffer
+              );
+              resetErrorOnSlide(value, 'max');
+              setMaxVal(value);
+              maxValRef.current = value;
+            }}
+            className={`thumb ${thumbPosition} z-400`}
+            style={{
+              zIndex: minVal <= max - 10 * minMaxBuffer ? '500' : '400'
+            }}
+          />
+          <div className='slider width-card position-relative height-3 display-flex flex-align-center'>
+            <div className='slider__track radius-md position-absolute height-05 z-100 width-full display-flex flex-align-center' />
+            <div
+              ref={range}
+              className='slider__range display-flex flex-align-center flex-justify-center radius-md position-absolute height-05 z-200'
+            >
+              {/* Show divergent map middle point */}
+              {min < 0 ? (
+                <div
+                  ref={middleMarker}
+                  className='position-relative height-2 width-1px middle-marker'
+                />
+              ) : null}
```

Comment:
How tricky would it be to have the `middleMarker` positioned in the center, at all times, even when dragging the thumbs? 

For example, I can see that the `middleMarker` is centered correctly within it's container, but in some cases (like in the screenshot) it looks misaligned because the left end of it's container is dynamically changed / set. Maybe tweaking the `left` and `width` values of the range ref could help? 

<img width="172" alt="Screenshot 2024-10-17 at 12 20 28" src="https://github.com/user-attachments/assets/41a64d86-8472-463f-b905-49b73bb48ce1">